src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -661,4 +661,11 @@
   <data name="SqlQueryProcessorRanOutOfInternalResourcesException" xml:space="preserve">
     <value>The query processor ran out of internal resources and could not produce a query plan. Please limit your parameters and retry.</value>
   </data>
-</root>
+  <data name="ResourceNotAllowedRestrictedByClinicalScopes" xml:space="preserve">
```

Comment:
Nit: Change to `ResourceTypeNotAllowedDuetoScope`, to use the same pattern as the error code below?